src/containerbuddy/config.go

```diff
@@ -245,14 +277,16 @@ func parseConfig(configFlag string) (*Config, error) {
 	} else {
 		data = []byte(configFlag)
 	}
+	return unmarshallConfig(data)
+}
 
+func unmarshallConfig(data []byte) (*Config, error) {
```

Comment:
The word "unmarshal" has 1 "L" not 2. You've got a double-l in a couple places in this PR.